dev-requirements.txt

```diff
@@ -2,7 +2,8 @@
 # https://caremad.io/posts/2013/07/setup-vs-requirement/
 
 # It doesn't seem like pip honors `dependency_links` in setup.py, so install
-# any forked packages here first.
+# any forked packages here, prior to `-e .`
+-e git+https://github.com/jnm/geojson-rewind@0.1.1+py2.jnm#egg=geojson-rewind
 
 # Install this package
 -e .
```

Comment:
`pip` version 19, doesn't seem to like this. 
> ERROR: Complete output from command python setup.py egg_info:
   ERROR: error in formpack setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers

Reply:
It works okay for me, but we should troubleshoot further. I've ~~attached~~ pasted, because GitHub won't let me upload the file, the full terminal transcript for my installation.
```
john@scrappy:~/Documents/kobo/MapX GeoJSON/formpack$ rm -r ../../dockdev/packenv/
john@scrappy:~/Documents/kobo/MapX GeoJSON/formpack$ virtualenv ../../dockdev/packenv/
Running virtualenv with interpreter /usr/bin/python2
New python executable in /home/john/Documents/kobo/dockdev/packenv/bin/python2
Also creating executable in /home/john/Documents/kobo/dockdev/packenv/bin/python
Installing setuptools, pkg_resources, pip, wheel...done.
john@scrappy:~/Documents/kobo/MapX GeoJSON/formpack$ . ../../dockdev/packenv/bin/activate
(packenv) john@scrappy:~/Documents/kobo/MapX GeoJSON/formpack$ pip install -r dev-requirements.txt 
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Obtaining file:///home/john/Documents/kobo/MapX%20GeoJSON/formpack (from -r dev-requirements.txt (line 9))
Obtaining geojson-rewind from git+https://github.com/jnm/geojson-rewind@0.1.1+py2.jnm#egg=geojson-rewind (from -r dev-requirements.txt (line 6))
  Cloning https://github.com/jnm/geojson-rewind (to revision 0.1.1+py2.jnm) to /home/john/Documents/kobo/dockdev/packenv/src/geojson-rewind
  Running command git clone -q https://github.com/jnm/geojson-rewind /home/john/Documents/kobo/dockdev/packenv/src/geojson-rewind
Collecting coverage==4.5.2 (from -r dev-requirements.txt (line 12))
  Using cached https://files.pythonhosted.org/packages/af/cf/e240715a50d64b0538a08a2be1ae4060fb25791e911cbe8fd2367b5deeaa/coverage-4.5.2-cp27-cp27mu-manylinux1_x86_64.whl
Collecting nose==1.3.7 (from -r dev-requirements.txt (line 13))
  Using cached https://files.pythonhosted.org/packages/99/4f/13fb671119e65c4dce97c60e67d3fd9e6f7f809f2b307e2611f4701205cb/nose-1.3.7-py2-none-any.whl
Collecting tox==3.7.0 (from -r dev-requirements.txt (line 14))
  Using cached https://files.pythonhosted.org/packages/4a/bd/50cd91221508db5bb66e211ac327d0da4a5db6d0db5ad35d6fbf1e4af5dc/tox-3.7.0-py2.py3-none-any.whl
Collecting flake8==3.6.0 (from -r dev-requirements.txt (line 15))
  Using cached https://files.pythonhosted.org/packages/34/a6/49e2849a0e5464e1b5d621f63bc8453066f0f367bb3b744a33fca0bc1ddd/flake8-3.6.0-py2.py3-none-any.whl
Collecting pytest==4.3.1 (from -r dev-requirements.txt (line 16))
  Using cached https://files.pythonhosted.org/packages/c8/52/c455e718f19e4fd7126c81b4910ca2431ded6c28b97e36a1a1f5e5ef7247/pytest-4.3.1-py2.py3-none-any.whl
Collecting funcsigs==1.0.2 (from -r dev-requirements.txt (line 17))
  Using cached https://files.pythonhosted.org/packages/69/cb/f5be453359271714c01b9bd06126eaf2e368f1fddfff30818754b5ac2328/funcsigs-1.0.2-py2.py3-none-any.whl
Collecting pytest-cov==2.6.1 (from -r dev-requirements.txt (line 18))
  Using cached https://files.pythonhosted.org/packages/7d/b5/92f32674ab954f80499ac73347bfeb815545ea295439c12b0ef3ac8f0975/pytest_cov-2.6.1-py2.py3-none-any.whl
Collecting coveralls==1.5.1 (from -r dev-requirements.txt (line 19))
  Using cached https://files.pythonhosted.org/packages/91/4f/f52682bfaa960445af93143df5ea0a98854f1a7829ff3e377932426bf48c/coveralls-1.5.1-py2.py3-none-any.whl
Collecting begins (from formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/ea/cf/6082c84e98e130e60df7d7bfe4bbb2cf020da230428554bab6e2d1469f34/begins-0.9-py2-none-any.whl
Collecting cyordereddict (from formpack==1.4->-r dev-requirements.txt (line 9))
Collecting jsonschema (from formpack==1.4->-r dev-requirements.txt (line 9))
  Downloading https://files.pythonhosted.org/packages/54/48/f5f11003ceddcd4ad292d4d9b5677588e9169eef41f88e38b2888e7ec6c4/jsonschema-3.0.2-py2.py3-none-any.whl (54kB)
     |████████████████████████████████| 61kB 580kB/s 
Collecting lxml (from formpack==1.4->-r dev-requirements.txt (line 9))
  Downloading https://files.pythonhosted.org/packages/e4/f4/65d145cd6917131826050b0479be35aaccba2847b7f80fc4afc6bec6616b/lxml-4.4.1-cp27-cp27mu-manylinux1_x86_64.whl (5.7MB)
     |████████████████████████████████| 5.7MB 14kB/s 
Collecting path.py<12 (from formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/1e/4a/6e20d9e264d2eb4977b94ee2f189d5c4a4819239ce063027a952014b6670/path.py-11.5.2-py2.py3-none-any.whl
Collecting pyquery (from formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/09/c7/ce8c9c37ab8ff8337faad3335c088d60bed4a35a4bed33a64f0e64fbcf29/pyquery-1.4.0-py2.py3-none-any.whl
Collecting pyxform (from formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/8c/ee/e27ecd5618a5abc3296d15a299b8725834cc8c3479adca9327891033d659/pyxform-0.15.0-py2.py3-none-any.whl
Collecting statistics (from formpack==1.4->-r dev-requirements.txt (line 9))
Collecting XlsxWriter (from formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/3f/1c/d6d90eb4e94b32b8558296ef197445fb1faca71d747e28ee3ef56f2cfac2/XlsxWriter-1.1.8-py2.py3-none-any.whl
Collecting backports.csv (from formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/8e/26/a6bd68f13e0f38fbb643d6e497fc3462be83a0b6c4d43425c78bb51a7291/backports.csv-1.0.7-py2.py3-none-any.whl
Collecting filelock<4,>=3.0.0 (from tox==3.7.0->-r dev-requirements.txt (line 14))
Collecting six<2,>=1.0.0 (from tox==3.7.0->-r dev-requirements.txt (line 14))
  Using cached https://files.pythonhosted.org/packages/73/fb/00a976f728d0d1fecfe898238ce23f502a721c0ac0ecfedb80e0d88c64e9/six-1.12.0-py2.py3-none-any.whl
Collecting virtualenv>=1.11.2 (from tox==3.7.0->-r dev-requirements.txt (line 14))
  Downloading https://files.pythonhosted.org/packages/db/9e/df208b2baad146fe3fbe750eacadd6e49bcf2f2c3c1117b7192a7b28aec4/virtualenv-16.7.2-py2.py3-none-any.whl (3.3MB)
     |████████████████████████████████| 3.3MB 736kB/s 
Collecting pluggy<1,>=0.3.0 (from tox==3.7.0->-r dev-requirements.txt (line 14))
  Using cached https://files.pythonhosted.org/packages/06/ee/de89e0582276e3551df3110088bf20844de2b0e7df2748406876cc78e021/pluggy-0.12.0-py2.py3-none-any.whl
Requirement already satisfied: setuptools>=30.0.0 in /home/john/Documents/kobo/dockdev/packenv/lib/python2.7/site-packages (from tox==3.7.0->-r dev-requirements.txt (line 14)) (41.1.0)
Collecting py<2,>=1.4.17 (from tox==3.7.0->-r dev-requirements.txt (line 14))
  Using cached https://files.pythonhosted.org/packages/76/bc/394ad449851729244a97857ee14d7cba61ddb268dce3db538ba2f2ba1f0f/py-1.8.0-py2.py3-none-any.whl
Collecting toml>=0.9.4 (from tox==3.7.0->-r dev-requirements.txt (line 14))
  Using cached https://files.pythonhosted.org/packages/a2/12/ced7105d2de62fa7c8fb5fce92cc4ce66b57c95fb875e9318dba7f8c5db0/toml-0.10.0-py2.py3-none-any.whl
Collecting configparser; python_version < "3.2" (from flake8==3.6.0->-r dev-requirements.txt (line 15))
  Using cached https://files.pythonhosted.org/packages/ab/1a/ec151e5e703ac80041eaccef923611bbcec2b667c20383655a06962732e9/configparser-3.8.1-py2.py3-none-any.whl
Collecting pyflakes<2.1.0,>=2.0.0 (from flake8==3.6.0->-r dev-requirements.txt (line 15))
  Using cached https://files.pythonhosted.org/packages/44/98/af7a72c9a543b1487d92813c648cb9b9adfbc96faef5455d60f4439aa99b/pyflakes-2.0.0-py2.py3-none-any.whl
Collecting enum34; python_version < "3.4" (from flake8==3.6.0->-r dev-requirements.txt (line 15))
  Using cached https://files.pythonhosted.org/packages/c5/db/e56e6b4bbac7c4a06de1c50de6fe1ef3810018ae11732a50f15f62c7d050/enum34-1.1.6-py2-none-any.whl
Collecting mccabe<0.7.0,>=0.6.0 (from flake8==3.6.0->-r dev-requirements.txt (line 15))
  Using cached https://files.pythonhosted.org/packages/87/89/479dc97e18549e21354893e4ee4ef36db1d237534982482c3681ee6e7b57/mccabe-0.6.1-py2.py3-none-any.whl
Collecting pycodestyle<2.5.0,>=2.4.0 (from flake8==3.6.0->-r dev-requirements.txt (line 15))
  Using cached https://files.pythonhosted.org/packages/e5/c6/ce130213489969aa58610042dff1d908c25c731c9575af6935c2dfad03aa/pycodestyle-2.4.0-py2.py3-none-any.whl
Collecting pathlib2>=2.2.0; python_version < "3.6" (from pytest==4.3.1->-r dev-requirements.txt (line 16))
  Using cached https://files.pythonhosted.org/packages/67/c6/4dbf5dfdbe1140cadf765c3896acc098578626c35721bc7d3eb35f6a8fc1/pathlib2-2.3.4-py2.py3-none-any.whl
Collecting attrs>=17.4.0 (from pytest==4.3.1->-r dev-requirements.txt (line 16))
  Using cached https://files.pythonhosted.org/packages/23/96/d828354fa2dbdf216eaa7b7de0db692f12c234f7ef888cc14980ef40d1d2/attrs-19.1.0-py2.py3-none-any.whl
Collecting more-itertools<6.0.0,>=4.0.0; python_version <= "2.7" (from pytest==4.3.1->-r dev-requirements.txt (line 16))
  Using cached https://files.pythonhosted.org/packages/2f/9d/dcfe59e213093695f108508af1214cf9cd95cc5489e46877ec5cb56369e5/more_itertools-5.0.0-py2-none-any.whl
Collecting atomicwrites>=1.0 (from pytest==4.3.1->-r dev-requirements.txt (line 16))
  Using cached https://files.pythonhosted.org/packages/52/90/6155aa926f43f2b2a22b01be7241be3bfd1ceaf7d0b3267213e8127d41f4/atomicwrites-1.3.0-py2.py3-none-any.whl
Collecting requests>=1.0.0 (from coveralls==1.5.1->-r dev-requirements.txt (line 19))
  Using cached https://files.pythonhosted.org/packages/51/bd/23c926cd341ea6b7dd0b2a00aba99ae0f828be89d72b2190f27c11d4b7fb/requests-2.22.0-py2.py3-none-any.whl
Collecting urllib3[secure]; python_version < "3" (from coveralls==1.5.1->-r dev-requirements.txt (line 19))
  Using cached https://files.pythonhosted.org/packages/e6/60/247f23a7121ae632d62811ba7f273d0e58972d75e58a94d329d51550a47d/urllib3-1.25.3-py2.py3-none-any.whl
Collecting docopt>=0.6.1 (from coveralls==1.5.1->-r dev-requirements.txt (line 19))
Collecting pyrsistent>=0.14.0 (from jsonschema->formpack==1.4->-r dev-requirements.txt (line 9))
  Downloading https://files.pythonhosted.org/packages/b9/66/b2638d96a2d128b168d0dba60fdc77b7800a9b4a5340cefcc5fc4eae6295/pyrsistent-0.15.4.tar.gz (107kB)
     |████████████████████████████████| 112kB 132kB/s 
Collecting functools32; python_version < "3" (from jsonschema->formpack==1.4->-r dev-requirements.txt (line 9))
Collecting importlib-metadata>=0.5 (from path.py<12->formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/ad/aa/25fcbded2ab4ed4ff3071d1e000cd4f8f9c65653d2d7157dd105a8e81d42/importlib_metadata-0.19-py2.py3-none-any.whl
Collecting backports.os; python_version == "2.7" and sys_platform == "linux2" (from path.py<12->formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/20/c2/193ff30cab747f205535a3a8c24fcc7647dbaf0aa667b7dd580a740b8de4/backports.os-0.1.1-py2-none-any.whl
Collecting cssselect>0.7.9 (from pyquery->formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/3b/d4/3b5c17f00cce85b9a1e6f91096e1cc8e8ede2e1be8e96b87ce1ed09e92c5/cssselect-1.1.0-py2.py3-none-any.whl
Collecting xlrd>=1.1.0 (from pyxform->formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/b0/16/63576a1a001752e34bf8ea62e367997530dc553b689356b9879339cf45a4/xlrd-1.2.0-py2.py3-none-any.whl
Collecting unittest2 (from pyxform->formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/72/20/7f0f433060a962200b7272b8c12ba90ef5b903e218174301d0abfd523813/unittest2-1.1.0-py2.py3-none-any.whl
Collecting unicodecsv>=0.14.1 (from pyxform->formpack==1.4->-r dev-requirements.txt (line 9))
Collecting formencode (from pyxform->formpack==1.4->-r dev-requirements.txt (line 9))
Collecting docutils>=0.3 (from statistics->formpack==1.4->-r dev-requirements.txt (line 9))
  Downloading https://files.pythonhosted.org/packages/3a/dc/bf2b15d1fa15a6f7a9e77a61b74ecbbae7258558fcda8ffc9a6638a6b327/docutils-0.15.2-py2-none-any.whl (548kB)
     |████████████████████████████████| 552kB 84kB/s 
Collecting scandir; python_version < "3.5" (from pathlib2>=2.2.0; python_version < "3.6"->pytest==4.3.1->-r dev-requirements.txt (line 16))
Collecting certifi>=2017.4.17 (from requests>=1.0.0->coveralls==1.5.1->-r dev-requirements.txt (line 19))
  Using cached https://files.pythonhosted.org/packages/69/1b/b853c7a9d4f6a6d00749e94eb6f3a041e342a885b87340b79c1ef73e3a78/certifi-2019.6.16-py2.py3-none-any.whl
Collecting chardet<3.1.0,>=3.0.2 (from requests>=1.0.0->coveralls==1.5.1->-r dev-requirements.txt (line 19))
  Using cached https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl
Collecting idna<2.9,>=2.5 (from requests>=1.0.0->coveralls==1.5.1->-r dev-requirements.txt (line 19))
  Using cached https://files.pythonhosted.org/packages/14/2c/cd551d81dbe15200be1cf41cd03869a46fe7226e7450af7a6545bfc474c9/idna-2.8-py2.py3-none-any.whl
Collecting pyOpenSSL>=0.14; extra == "secure" (from urllib3[secure]; python_version < "3"->coveralls==1.5.1->-r dev-requirements.txt (line 19))
  Using cached https://files.pythonhosted.org/packages/01/c8/ceb170d81bd3941cbeb9940fc6cc2ef2ca4288d0ca8929ea4db5905d904d/pyOpenSSL-19.0.0-py2.py3-none-any.whl
Collecting cryptography>=1.3.4; extra == "secure" (from urllib3[secure]; python_version < "3"->coveralls==1.5.1->-r dev-requirements.txt (line 19))
  Using cached https://files.pythonhosted.org/packages/e6/68/50698ce24c61db7d44d93a5043c621a0ca7839d4ef9dff913e6ab465fc92/cryptography-2.7-cp27-cp27mu-manylinux1_x86_64.whl
Collecting ipaddress; python_version == "2.7" and extra == "secure" (from urllib3[secure]; python_version < "3"->coveralls==1.5.1->-r dev-requirements.txt (line 19))
  Using cached https://files.pythonhosted.org/packages/fc/d0/7fc3a811e011d4b388be48a0e381db8d990042df54aa4ef4599a31d39853/ipaddress-1.0.22-py2.py3-none-any.whl
Collecting zipp>=0.5 (from importlib-metadata>=0.5->path.py<12->formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/da/bd/1a5fdf15aa44231fd09f63ecf175b60f057ae37ec65b343bb009364923f3/zipp-0.5.2-py2.py3-none-any.whl
Collecting contextlib2; python_version < "3" (from importlib-metadata>=0.5->path.py<12->formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/a2/71/8273a7eeed0aff6a854237ab5453bc9aa67deb49df4832801c21f0ff3782/contextlib2-0.5.5-py2.py3-none-any.whl
Collecting future (from backports.os; python_version == "2.7" and sys_platform == "linux2"->path.py<12->formpack==1.4->-r dev-requirements.txt (line 9))
Requirement already satisfied: argparse in /usr/lib/python2.7 (from unittest2->pyxform->formpack==1.4->-r dev-requirements.txt (line 9)) (1.2.1)
Collecting traceback2 (from unittest2->pyxform->formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/17/0a/6ac05a3723017a967193456a2efa0aa9ac4b51456891af1e2353bb9de21e/traceback2-1.4.0-py2.py3-none-any.whl
Collecting asn1crypto>=0.21.0 (from cryptography>=1.3.4; extra == "secure"->urllib3[secure]; python_version < "3"->coveralls==1.5.1->-r dev-requirements.txt (line 19))
  Using cached https://files.pythonhosted.org/packages/ea/cd/35485615f45f30a510576f1a56d1e0a7ad7bd8ab5ed7cdc600ef7cd06222/asn1crypto-0.24.0-py2.py3-none-any.whl
Collecting cffi!=1.11.3,>=1.8 (from cryptography>=1.3.4; extra == "secure"->urllib3[secure]; python_version < "3"->coveralls==1.5.1->-r dev-requirements.txt (line 19))
  Using cached https://files.pythonhosted.org/packages/8d/e9/0c8afd1579e5cf7bc0f06fbcd7cdb954cbc0baadd505973949a99337da1c/cffi-1.12.3-cp27-cp27mu-manylinux1_x86_64.whl
Collecting linecache2 (from traceback2->unittest2->pyxform->formpack==1.4->-r dev-requirements.txt (line 9))
  Using cached https://files.pythonhosted.org/packages/c7/a3/c5da2a44c85bfbb6eebcfc1dde24933f8704441b98fdde6528f4831757a6/linecache2-1.0.0-py2.py3-none-any.whl
Collecting pycparser (from cffi!=1.11.3,>=1.8->cryptography>=1.3.4; extra == "secure"->urllib3[secure]; python_version < "3"->coveralls==1.5.1->-r dev-requirements.txt (line 19))
Building wheels for collected packages: pyrsistent
  Building wheel for pyrsistent (setup.py) ... done
  Created wheel for pyrsistent: filename=pyrsistent-0.15.4-cp27-cp27mu-linux_x86_64.whl size=95648 sha256=d12b596a3f7229056968ee928980bab29be957724d0b0e30dfeeddfbddd8c21b
  Stored in directory: /home/john/.cache/pip/wheels/bb/46/00/6d471ef0b813e3621f0abe6cb723c20d529d39a061de3f7c51
Successfully built pyrsistent
Installing collected packages: geojson-rewind, coverage, nose, filelock, six, virtualenv, zipp, configparser, contextlib2, scandir, pathlib2, importlib-metadata, pluggy, py, toml, tox, pyflakes, enum34, mccabe, pycodestyle, flake8, funcsigs, attrs, more-itertools, atomicwrites, pytest, pytest-cov, asn1crypto, ipaddress, pycparser, cffi, cryptography, pyOpenSSL, idna, certifi, urllib3, chardet, requests, docopt, coveralls, begins, cyordereddict, pyrsistent, functools32, jsonschema, lxml, future, backports.os, path.py, cssselect, pyquery, xlrd, linecache2, traceback2, unittest2, unicodecsv, formencode, pyxform, docutils, statistics, XlsxWriter, backports.csv, formpack
  Running setup.py develop for geojson-rewind
  Running setup.py develop for formpack
Successfully installed XlsxWriter-1.1.8 asn1crypto-0.24.0 atomicwrites-1.3.0 attrs-19.1.0 backports.csv-1.0.7 backports.os-0.1.1 begins-0.9 certifi-2019.6.16 cffi-1.12.3 chardet-3.0.4 configparser-3.8.1 contextlib2-0.5.5 coverage-4.5.2 coveralls-1.5.1 cryptography-2.7 cssselect-1.1.0 cyordereddict-1.0.0 docopt-0.6.2 docutils-0.15.2 enum34-1.1.6 filelock-3.0.12 flake8-3.6.0 formencode-1.3.1 formpack funcsigs-1.0.2 functools32-3.2.3.post2 future-0.17.1 geojson-rewind idna-2.8 importlib-metadata-0.19 ipaddress-1.0.22 jsonschema-3.0.2 linecache2-1.0.0 lxml-4.4.1 mccabe-0.6.1 more-itertools-5.0.0 nose-1.3.7 path.py-11.5.2 pathlib2-2.3.4 pluggy-0.12.0 py-1.8.0 pyOpenSSL-19.0.0 pycodestyle-2.4.0 pycparser-2.19 pyflakes-2.0.0 pyquery-1.4.0 pyrsistent-0.15.4 pytest-4.3.1 pytest-cov-2.6.1 pyxform-0.15.0 requests-2.22.0 scandir-1.10.0 six-1.12.0 statistics-1.0.3.5 toml-0.10.0 tox-3.7.0 traceback2-1.4.0 unicodecsv-0.14.1 unittest2-1.1.0 urllib3-1.25.3 virtualenv-16.7.2 xlrd-1.2.0 zipp-0.5.2
(packenv) john@scrappy:~/Documents/kobo/MapX GeoJSON/formpack$ pip --version
ppip 19.2.2 from /home/john/Documents/kobo/dockdev/packenv/local/lib/python2.7/site-packages/pip (python 2.7)
(packenv) john@scrappy:~/Documents/kobo/MapX GeoJSON/formpack$ python --version
Python 2.7.15+
```

Reply:
Found the issue on my side. 
In my environment, `python3` is the default one. When I (re)created the virtualenv, I didn't specify I wanted and I didn't notice it picked `python3` by default. Something weird, `pytest` was reporting `python 2.7.15+`. (maybe related to what you said earlier).
After having recreated the venv correctly, everything is installed as expected.